build-tools/packages/build-cli/src/library/index.ts

```diff
@@ -60,3 +60,4 @@ export {
 } from "./release.js";
 export { LayerGraph } from "./layerGraph.js";
 export { type Handler, policyHandlers } from "./repoPolicyCheck/index.js";
+export { Node10CompatExportData } from "./packageExports.js";
```

Comment:
As in the other PR, just import directly from the submodules and remove the export here.